snippets/src/language/normalize_code.rs

```diff
@@ -0,0 +1,56 @@
+use std::borrow::Cow;
+
+use super::{normalize_comments, snippet_context::SnippetContext};
+use crate::text::normalize_space;
+
+/// Perform both comment and code normalization.
+///
+/// Uses [`super::normalize_comments`] and [`crate::text::normalize_space`].
+#[tracing::instrument(skip_all)]
+pub fn normalize_code<'a>(context: &'a SnippetContext) -> Cow<'a, [u8]> {
+    let comments = normalize_comments(context);
```

Comment:
[nit] [pedantic] This variable should be `without_comments` or `uncommented` or something like that

Reply:
I turns out I don't need a variable at all, so I've removed it.